velox/docs/functions/window.rst

```diff
@@ -82,3 +82,9 @@ Ranking functions
 
 Returns a unique, sequential number for each row, starting with one, according to the ordering of rows
 within the window partition.
+
+.. function:: rank() -> bigint
```

Comment:
Make sure to list functions in alphabetical order. rank would come before row_number.